src/aind_registration_evaluation/util/extract_roi.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+from skimage import io, filters, morphology, exposure, measure
+import matplotlib.pyplot as plt
+import tifffile as tiff
+from sklearn.metrics import mutual_info_score, normalized_mutual_info_score
+from scipy.stats import entropy
+
+def get_props(image, binaryThreshold):
+    '''
+    	- Applies Gaussian blur to smooth the image
```

Comment:
Please, make sure all the documentation follows the numpy style docstrings structure.